README.md

```diff
@@ -1,4 +1,6 @@
 # contact-manager
+[![JavaRush University](https://img.shields.io/badge/java-rush-orange?style=for-the-badge&color=orange&cacheSeconds=3600)](https://javarush.com/ua/university/)
+
```

Comment:
This should not be here :D 
I suppose you create this branch from your prev branch. Please, create a new branch only from `main`

Reply:
Ohh, maybe now I get it. I see what happened. You updated this branch by merging main into it. It's fine then.